apps/css-workshop/src/components/Anchor.astro

```diff
@@ -0,0 +1,18 @@
+---
+type Props = {
+  external?: 'true';
+  status?: 'informational' | 'positive' | 'warning' | 'negative';
+  underline?: 'true';
+} & astroHTML.JSX.HTMLAttributes;
+
+const { class: className, external, href, status, underline, ...props } = Astro.props;
+---
+
+<a
+  class:list={['iui-anchor', className, external ? 'iui-anchor-external' : '']}
+  data-iui-status={status}
+  data-iui-underline={underline}
+  {...external ? { referrerpolicy: 'no-referrer', target: '_blank' } : {}}
```

Comment:
`referrerpolicy` and `target` should be added by the consumer, rather than being handled in here.

Reply:
Removed from the component, inlined on anchor demo page.